test/helpers/testabilityPatch.js

```diff
@@ -381,6 +381,9 @@ function generateInputCompilerHelper(helper) {
         // Compile the lot and return the input element
         $compile(helper.formElm)(scope);
 
+        $rootElement.append(helper.formElm);
+        jqLite($document[0].body).append($rootElement);
```

Comment:
OK, I see why it is needed. Is it necessary on all browsers or some of them ? Which ones ?